defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+# Service accounts
+monitoring_sa: monitoring
```

Comment:
I wouldn't specify any of those sa names explicitly here

Reply:
Fixed